.NET/Microsoft.Recognizers.Text.Sequence/Constants.cs

```diff
@@ -9,6 +9,12 @@ public static class Constants
         // Model type name
         public const string MODEL_PHONE_NUMBER = "phonenumber";
 
+        public const string MODEL_IP = "ip";
```

Comment:
Why 3 models? Shouldn't it be just one with different subtypes?

Reply:
fixed

Reply:
Thanks for your review. Some comments are missed when I try to rename the file, could you help to do it again?